scripts/get-oa-unpaywall-data-tv.R

```diff
@@ -0,0 +1,115 @@
+# Get open access data from the Unpaywall API for new/changed pubs in TV
+# Added 10.1038/nature24628 for NCT02509962
+# Change 10.1016/j.euroneuro.2015.04.002 for NCT01266655
```

Comment:
To clarify, right now just **adding** the 2nd DOI and not **removing** the one it will replace, correct?

Reply:
@maia-sh, yup! This way we still have the option to check improvement on the non-earliest publication, if we want to.

Reply:
ok, would say "add" instead of "change" but nbd